### ISSUE TITLE
Accessibility improvements

### ISSUE THREAD

**Issue by Denperidge** [Denperidge-Redpencil/semantic.works#5]
Things to look out for:
- [x] Tab navigation
- [x] Tab navigation highlights
- [x] Alt text
- [x] Semantic HTML
- [x] Research ARIA

**Comment by Denperidge** [comment]
Added a good amount of semantic html in a7d9fd0ba993796006d82e5821f768b95ddf2efc

**Comment by Denperidge** [comment]
Added alt text in 2abce45fa66f1355c4cbdfff2e3107b40012f336

**Comment by Denperidge** [comment]
Fixed tab navigation highlights in dcbcf02788750c5ec0f9f60cbdb8f7568658c35f !

**Comment by Denperidge** [comment]
It should be okay at this point! Although a few tests should be run down the road, the basic accessibility is a *lot* better now
